modules/openapi-generator/src/main/resources/python/python-experimental/api_client.mustache

```diff
@@ -526,7 +526,11 @@ class ApiClient(object):
         """
         if not auth_settings:
             return
-
+        if self.configuration.security_schemes:
+            auth_settings = [
+                value for value in auth_settings
+                if value in self.configuration.security_schemes
+            ]
         for auth in auth_settings:
             auth_setting = self.configuration.auth_settings().get(auth)
             if auth_setting:
```

Comment:
Now that we will be returning only auth_settings that exist, we can remove the:
`if not auth_settings['value']` code here.
Per its history: https://github.com/OpenAPITools/openapi-generator/commit/271661aec79771e15ce65943a1ed1e8af1ca4d34
It was only added to skip over handling an auth_setting where values were empty.
The proposed addition will no longer return an empty auth_setting, so we can remove it.

Reply:
I have removed the test.